editors/sc-ide/widgets/help_browser.cpp

```diff
@@ -114,6 +109,11 @@ HelpBrowser::HelpBrowser(QWidget* parent): QWidget(parent) {
 }
 
 void HelpBrowser::onPageLoad() {
+    // add these actions to weview's renderer, to capture shift+enter and possibly other swallowed shortcuts
+    ((OverridingAction*)mActions[EvaluateRegion])->addToWidget(mWebView->focusProxy());
+    ((OverridingAction*)mActions[Evaluate])->addToWidget(mWebView->focusProxy());
+
+    // unused: see HelpSource/scdoc.js and HelpSource/editor.js
```

Comment:
is this comment saying all the webchannel/websocket stuff is unused now? if so, delete it, it's still there in source control